sample/SampleServer/Program.cs

```diff
@@ -28,10 +28,9 @@ static void Main(string[] args)
                     .WithLoggerFactory(new LoggerFactory())
                     .AddDefaultLoggingProvider()
                     .WithMinimumLogLevel(LogLevel.Trace)
+                    .WithHandler<TextDocumentHandler>()
```

Comment:
BTW, looks to me like this overload will auto-register any JSON-RPC interfaces implemented by the handler type. Is that correct?

Reply:
yeah, the handler collection will create a descriptor per `IJsonRpcHandler` interface it finds in the implementing class.